pkgs/development/python-modules/coordinates/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "coordinates";
+  version = "0.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1ynsrymapazhs03p6yxbcdhbq5ygdwdnkzab7jmpp6whhd0kf520";
+  };
+
+  # No tests in PyPi source distribution.
+  doCheck = false;
```

Comment:
I see you've requested these be bundled with the PyPi dist, but possibly consider switching to the GitHub release for now?